ooc-cli.use

```diff
@@ -0,0 +1,6 @@
+Name: ooc CLI
+Description: CLI helpers
+SourcePath: source/cli
+Requires: ooc-collections
```

Comment:
Wouldn't it be better if the requires line reflected the requirement in that component. We should never have a use in the code of a component that is not in the `Requires:` line of its use-file.